### ISSUE TITLE
Error install Parrot OS

### ISSUE THREAD

**Issue by azharrafif5** [calamares/calamares#1704]
Help,  I get error while installing Parrot OS

![Screenshot at 2021-05-24 14-17-54](https://user-images.githubusercontent.com/75826509/119281636-a2414180-bc60-11eb-8c6b-4850eb21c764.png)

please, I want to use Parrot OS

**Comment by monetaryabyss** [comment]
The best way for us to help you would be for us to look at the logs (normally `~/.cache/calamares/session.log` ) on the system where this error was produced and the settings (starting with `settings.conf` is a good start but in this case `finished.conf` would also be helpful) that you have used. You can use a paste service like [dpaste](https://dpaste.org), [paste.debian](http://paste.debian.net), [paste.ubuntu](https://paste.ubuntu.com) as well as many others. Just provide us with a link to those in a comment when you have them. If you start Calamares with -D6 appended usually provides a useful log output.
In other words, we are missing a lot of detail needed to help you troubleshoot your issue. A screenshot is insufficient to be able to determine the problem in most cases.

**Comment by kkofler** [comment]
As far as I can tell from the screenshot, this is `mkinitramfs` failing, not a Calamares issue at all. If a CLI tool called by Calamares fails with an error, Calamares can only report that error, there's not much more it can do.

**Comment by demmm** [comment]
For the forth time @azharrafif5 , this is not a help channel. Bug reports are for **BUG** reports.
Again, nothing is done according to the ISSUE_TEMPLATE which you continually choose to ignore. For help visit IRC or the telegram channel.
